src/main/java/com/jalasoft/webservice/model/ImageConvert.java

```diff
@@ -61,13 +52,13 @@ public String ImageConvert(String sourcePath, int dpi, String destinationPath, S
 
             //Instantiating the PDFRenderer class
             PDFRenderer renderer = new PDFRenderer(document);
-            for (int page = 0; page < document.getNumberOfPages(); ++page){
+            for (int page = 0; page < document.getNumberOfPages(); ++page) {
 
                 //Rendering the image into a buffer image
                 BufferedImage img = renderer.renderImageWithDPI(page, dpis, ImageType.RGB);
 
                 //Creating images files
-                String fileName =  destination + page + ext;
+                String fileName = destination + page + ext;
```

Comment:
It is not necessary to explain every line because the others reviewers are also programmers.

Reply:
Can you review the new commit with the suggestion changes?